.zuul.yaml

```diff
@@ -2,20 +2,20 @@
 - project:
     name: openstack-k8s-operators/watcher-operator
     default-branch: main
+    templates:
+      - opendev-master-watcher-operator-pipeline
     github-check:
       jobs:
         - noop
         - watcher-operator-doc-preview
-        - openstack-meta-content-provider:
-            vars:
-              cifmw_operator_build_meta_build: false
         - watcher-operator-validation
         - watcher-operator-kuttl
 
 - job:
     name: watcher-operator-base
+    override-checkout: main
     parent: podified-multinode-edpm-deployment-crc-2comp
-    dependencies: ["openstack-meta-content-provider"]
+    dependencies: ["openstack-meta-content-provider-master"]
```

Comment:
why have the dependency at all then if we aren't going to use the built containers. do we need it for something else?